### DISCUSSION TITLE
How To Set TicksPaint

### DISCUSSION THREAD

**Discussion by ajayre** [Live-Charts/LiveCharts2#628] [General]
WinForms and Cartesian chart. I don't seem to have access to the TicksPaint property. I'm sure it's a cast I am missing but looking through the source code I cannot figure it out. Here is what I have so far.

`            Chart.XAxes = new LiveChartsCore.Axis<SkiaSharpDrawingContext, LabelGeometry, LineGeometry>[]
            {
                new LiveChartsCore.SkiaSharpView.Axis
                {
                    TextSize = 12,
                    Name = "Time (ms)",
                    NameTextSize = 12,
                }
            };
`

Thanks, Andy

**Comment by beto-rodriguez** [comment]
This is available only in the latest release (beta 400)

 https://github.com/beto-rodriguez/LiveCharts2/blob/master/samples/ViewModelsSamples/Axes/Style/ViewModel.cs
